src/iostream.c

```diff
@@ -211,7 +211,6 @@ static UInt OpenPty(int * master, int * slave)
         goto error;
     }
 
-    ttyname = ;
```

Comment:
What's going on here? Is this valid C? Is this ever compiled?

Reply:
The code was invalid, so nobody used it since I introduced it in April (in commit 138fc2b9c40ded7b88404cd39bee496ce1ff8662 but I am not sure how exactly this mixup happened. Ah well :-/).

One reason to keep this code is that the code *do* use on Linux, *BSD, OSX & Cygwin uses `openpty`, which is not part of POSIX; while this alternate code path is using only POSIX APIs. So it's arguably more portable. Of course we might not care about supporting other UNIX like operating systems. One may also rightfully argue that it's better to remove this dead code and wait for somebody to complain, and then add it back (but this time with an actual use case). I am all open for that.

Personally, I'd prefer to do this in a later PR, instead of delaying this one.